packages/perseus/src/widgets/definition/definition.tsx

```diff
@@ -92,10 +93,10 @@ class Definition extends React.Component<DefinitionProps> implements Widget {
 
 const styles = {
     tooltipBody: {
-        color: color.offBlack,
-        fontSize: 20,
-        fontWeight: 500,
-        lineHeight: "30px",
+        color: semanticColor.core.foreground.neutral.strong,
+        fontSize: "2rem",
```

Comment:
Still need to discuss this with our designer as this is one attribute that does have semantic tokens, just not ones that provide this value. Might need to add new token, change the design, or just go with this.